scripts/reportportal/rp_manual_reporter/cluster_info.py

```
@@ -0,0 +1,170 @@
+        api_url = infra.instance.status.apiServerURL
+        if api_url:
+            attrs.cluster_domain = _extract_domain_from_api_url(api_url=api_url)
+    except (AttributeError, KeyError, TypeError) as exc:
+        LOGGER.warning(f"Failed to get cluster infrastructure info: {exc}")
+
+    # Default storage class
+    try:
+        default_sc = get_default_storage_class(client=client)
+        attrs.storage_class = default_sc.name if default_sc else None
+    except (AttributeError, KeyError, TypeError) as exc:
+        LOGGER.warning(f"Failed to get default storage class: {exc}")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)